test/js/ez-navigation-hub.js

```diff
@@ -7,4 +12,128 @@ describe('ez-navigation-hub', function() {
             element.constructor
         );
     });
+    describe('properties', function () {
+        describe('`activeZoneClass`', function () {
+            it('should have a default value', function () {
+                assert.equal(element.activeZoneClass, element.getAttribute('active-zone-class'));
+            });
+
+            it('should set', function () {
+                const value = 'Something';
+
+                element.activeZoneClass = value;
```

Comment:
this is totally useless, you should set the value in the attribute and check that the property has the correct value.